Updating documentation instructions (#43)
aws-ia/terraform-repo-template: .header.md

## Module Documentation

> **Do not manually update README.md**. This file is automatically generated by pulling in content from other files. For instructions, including a fill-in-the-blank content template, see [Create readmes for Terraform-based Partner Solutions.](https://aws-ia-us-west-2.s3.us-west-2.amazonaws.com/docs/content/index.html#/lessons/8rpYWWL59M7dcS-NsjYmaISUu-L_UqEv)

Reviewer: ```suggestion
**Do not manually update README.md**. `terraform-docs` is used to generate README files by pulling in content from other files. For instructions, including a fill-in-the-blank content template, see [Create readmes for Terraform-based Partner Solutions.](https://aws-ia-us-west-2.s3.us-west-2.amazonaws.com/docs/content/index.html#/lessons/8rpYWWL59M7dcS-NsjYmaISUu-L_UqEv)
```

The `this file` statement might be confusing for some on the first read.

Reviewer 2: @tlindsay42 Thanks for the thoughtful suggestion. I wonder, though. Here's what I understand (and could be wrong). While we use the 'terraform-docs' command to generate the readme manually whenever we want to preview it, we don't have to use that command. The readme will automatically be updated whenever we commit changes to any of the source content (in HEADER.md or .tf files). Is that not true?

> **Do not manually update README.md**. This file is automatically generated by pulling in content from other files. For instructions, including a fill-in-the-blank content template, see [Create readmes for Terraform-based Partner Solutions.](https://aws-ia-us-west-2.s3.us-west-2.amazonaws.com/docs/content/index.html#/lessons/8rpYWWL59M7dcS-NsjYmaISUu-L_UqEv)

Reviewer 2: ```suggestion
**Do not manually update README.md**. README.md is automatically generated by pulling in content from other files. For instructions, including a fill-in-the-blank content template, see [Create readmes for Terraform-based Partner Solutions.](https://aws-ia-us-west-2.s3.us-west-2.amazonaws.com/docs/content/index.html#/lessons/8rpYWWL59M7dcS-NsjYmaISUu-L_UqEv)
```

Reviewer 2: @tlindsay42 Here's the edit that you and I agreed on in live conversation just now. Thanks!
